api/admin/google_oauth_admin_authentication_provider.py

```diff
@@ -18,15 +18,26 @@
 class GoogleOAuthAdminAuthenticationProvider(AdminAuthenticationProvider):
 
     NAME = ExternalIntegration.GOOGLE_OAUTH
-    DESCRIPTION = _("<p>Allow admins to sign in with their Google accounts.</p>" +
-                    "<p>To use this integration, visit <a target='_blank' href='https://console.developers.google.com/apis/dashboard'>the Google developer console.</a> " +
-                    "Create a project, click 'Create Credentials' in the left sidebar, and select 'OAuth client ID'. " +
-                    "If you get a warning about the consent screen, click 'Configure consent screen' and enter your library name as the product name. Save the consent screen information.</p>" +
-                    "<p>Choose 'Web Application' as the application type.</p>" +
-                    "<p>Leave 'Authorized JavaScript origins' blank, but under 'Authorized redirect URIs', add the url of your circulation manager followed by '/admin/GoogleAuth/callback', e.g. 'http://mycircmanager.org/admin/GoogleAuth/callback'.</p>" +
-                    "<p>Click create, and you'll get a popup with your new client ID and secret. Copy these values and enter them in the form below.</p>"),
+    DESCRIPTION = _("How to Configure a Google OAuth Integration")
     DOMAINS = "domains"
 
+    INSTRUCTIONS = _("<p>Configuring a Google OAuth integration in the Circulation Manager " +
+                    "will allow users to sign into the Admin interface with their Google/GMail credentials.</p>" +
```

Comment:
I might say "admins" instead of "users" here just so it's clear this isn't intended to allow anyone to log in, although it can be configured to allow anyone from a certain domain.